src/provider/states/starting.rs

```diff
@@ -36,6 +39,37 @@ impl State<PodState> for Starting {
                     );
                     return Transition::Complete(Err(enable_error));
                 }
+
+                let start_time = Instant::now();
+                // TODO: does this need to be configurable, or ar we happy with a hard coded value
```

Comment:
I'd say a livenessProbe could match: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command

That said: I'm fine with a hardcoded value for now

Reply:
I've added #55 for this.
Should probably not be necessary once we start subscribing to events for systemd units.